ivy/functional/frontends/tensorflow/keras/metrics.py

```diff
@@ -292,3 +292,36 @@ def cosine_similarity(y_true, y_pred):
 
     cosine = numerator / denominator
     return cosine
+
+
+# deserialize
+def deserialize(name, custom_objects=None):
+    return ivy.deserialize(name, custom_objects=custom_objects)
+
+
+deserialize.supported_dtypes = {
+    "numpy": (
+        "float16",
+        "float32",
+        "float64",
+    ),
+    "tensorflow": (
+        "float16",
+        "float32",
+        "float64",
+    ),
+    "torch": (
+        "float32",
+        "float64",
+    ),
+    "jax": (
+        "float16",
+        "float32",
+        "float64",
+    ),
+}
+
+
+# get 
+def get(name, custom_objects=None):
```

Comment:
From [Tensorflow docs](https://www.tensorflow.org/api_docs/python/tf/keras/metrics/get) this function doesn't have `custom_objects` argument

Reply:
Yes, correct. I have made the required change locally & it works 👍